flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/SubtaskCheckpointCoordinatorImpl.java

```diff
@@ -870,7 +870,8 @@ private static void logCheckpointProcessingDelay(CheckpointMetaData checkpointMe
         long delay = System.currentTimeMillis() - checkpointMetaData.getReceiveTimestamp();
         if (delay >= CHECKPOINT_EXECUTION_DELAY_LOG_THRESHOLD_MS) {
             LOG.warn(
-                    "Time from receiving all checkpoint barriers/RPC to executing it exceeded threshold: {}ms",
+                    "Time from receiving all checkpoint barriers/RPC for checkpoint id {} to executing it exceeded threshold: {}ms",
```

Comment:
```suggestion
                    "Time from receiving all checkpoint barriers/RPC for checkpoint {} to executing it exceeded threshold: {}ms",
```
Maybe just use 'checkpoint' as other log info.